types/react-table/index.d.ts

```diff
@@ -1,10 +1,6 @@
 // Type definitions for react-table 6.7
 // Project: https://github.com/react-tools/react-table
-// Definitions by: Roy Xue <https://github.com/royxue>
-//                 Pavel Sakalo <https://github.com/psakalo>
-//                 Krzysztof Porębski <https://github.com/Havret>
-//                 Grzegorz Rozdzialik <https://github.com/Gelio>
-//                 Andy S <https://github.com/andys8>
+// Definitions by: Roy Xue <https://github.com/royxue>, Pavel Sakalo <https://github.com/psakalo>, Krzysztof Porębski <https://github.com/Havret>, Andy S <https://github.com/andys8>
```

Comment:
Not sure why this change happened? Maybe vscode or dtslint did something automatic and I didn't notice?